### DISCUSSION TITLE
Add a resolve or retest button or link to the notification when a repair item is discovered

### DISCUSSION THREAD

**Discussion by yanky79** [frenck/spook#646] [Ideas]
I don't really know what to do after I repair an item, but the notification still remains.  The only option is to ignore - but I don't want to ignore it, I would like to have the issue either tested again or the ability to mark it as resolved.  It may be the same thing to ignore, but I don't think it is.  Right now, the only thing I know to do is to restart HA and that is pretty disruptive.  What to do isnt really covered in the documentation either.

**Comment by JiDW** [comment]
Hey, I have the same question. I'm wondering if I'm missing something.
Does anyone know if the issues are supposed to be detected as "repaired" after a while?
Thanks!

**Comment by frenck** [comment]
Thanks for the suggestion, but this suggestion is not the correct approach.

It should resolve itself; not using some button.

../Frenck

**Reply by yanky79** [reply]
The current experience, though, is the issue stays in the Notification until a HA reload or restart.

**Reply by frenck** [reply]
It should not be. Please report an issue in that case, with reproducible steps so it can be fixed.
